common/utils/src/main/resources/error/error-classes.json

```diff
@@ -657,6 +657,11 @@
           "The <exprName> must be between <valueRange> (current value = <currentValue>)."
         ]
       },
+      "WRONG_NUM_ARG_TYPES" : {
+        "message" : [
+          "The expression requires <expectedNum> argument types but the actual number is <actualNum>."
```

Comment:
I feel the number of argument types is a bit confusing to end users. shall we just say number of arguments?

Reply:
If we say that the number of arguments is wrong but actually the user passes correct arguments. That might confuse much more.